draft-ietf-quic-transport.md

```diff
@@ -831,29 +768,46 @@ larger resource commitments at the receiver.  Thus there is a trade-off between
 resource commitment and overhead when determining how large a limit is
 advertised.
 
-A receiver MAY use an autotuning mechanism to tune the frequency and amount that
-it increases data limits based on a round-trip time estimate and the rate at
-which the receiving application consumes data, similar to common TCP
-implementations.
+A receiver MAY use an autotuning mechanism to tune the frequency and amount of
+advertised additional credit.
```

Comment:
The cut here might be too much.  I'd say either cut the whole paragraph, or keep it.  I think keeping it is best.